apps/arena/lib/arena/game_updater.ex

```diff
@@ -317,6 +325,31 @@ defmodule Arena.GameUpdater do
     {:noreply, put_in(state, [:game_state, :status], :RUNNING)}
   end
 
+  def handle_info(:deathmatch_end_game_check, state) do
+    players =
+      state.game_state.players
+      |> Enum.map(fn {player_id, player} ->
+        %{kills: kills} = GameTracker.get_player_result(player_id)
+        {player_id, player, kills}
+      end)
+      |> Enum.sort_by(fn {_player_id, _player, kills} -> kills end, :desc)
+
+    {winner_id, winner, _kills} = Enum.at(players, 1)
```

Comment:
Shouldn't this be zero?
```suggestion
    {winner_id, winner, _kills} = Enum.at(players, 0)
```